app/Modules/MailQueue/Support/MailQueue.php

```diff
@@ -26,8 +26,8 @@ public function create($object, $input)
         return $object->mailQueue()->create([
             'from' => json_encode(['email' => $object->user->email, 'name' => $object->user->name]),
             'to' => json_encode($input['to']),
-            'cc' => json_encode(($input['cc']) ?: []),
-            'bcc' => json_encode(($input['bcc']) ?: []),
+            'cc' => json_encode(isset($input['cc']) ? $input['cc'] : []),

```

Comment:
Isn't that wrong? It sets the 'cc' field to $input['cc'] rather than the result of "json_encode($input['cc'])"...

I missed this PR and added my own in #635. Maybe you can work with that.

Reply:
Yeah you are right. I will check this again.